bin/quarry.lua

```diff
@@ -0,0 +1,193 @@
+local radius, keepItems = ...
+radius = tonumber(radius)
+
+if not radius then
+    print('Please specify a side length (number, arg 1)')
+    return
+end
+
+if keepItems == 'true' then
+    keepItems = true
+elseif keepItems == 'false' then
+    keepItems = false
+else
+    print('Please specify if I should keep junk items (e.g. cobblestone) (true/false, arg 2)')
+    return
+end
+
+if not turtle then
+    print('This must be run from a turtle')
+    return
+end
```

Comment:
this assertion should probably be moved sooner

Reply:
If you think it should be, feel free to, but I think it's fine